src/open_inwoner/components/templatetags/helpers.py

```diff
@@ -64,3 +64,22 @@ def as_attributes(attribute_dict):
     if not attribute_dict:
         return ""
     return format_html_join(" ", '{}="{}"', attribute_dict.items())
+
+
+@register.simple_tag
+def get_plan_participants(plan, user):
```

Comment:
Feel free to check with Bart, but I wouldn't use a templatetag for this, feels more like something you would set in your view(s)

Reply:
I kinda agree, I would put this in in get_context_data().

I'm not a big fan of template tags for many reasons, but also because they reduce visibility of what is going on. (especially database access and other IO). Sven & Jorik got a bit wild with them in this project.